fink_science/random_forest_snia/classifier_sigmoid.py

```diff
@@ -291,6 +291,29 @@ def get_train_test(percent_train):
 
     return sample
 
+def average_intraday_data(df_intra):
+    
+    """Average over intraday data points
+        
+     Parameters
+     ----------
+     df_intra: pd.DataFrame
+        containing the history of the flux
+        with intraday data
+
+     Returns
+     -------
+     df_average: pd.DataFrame
+        containing only daily data
+    
+    """
+    
+    df_average = df_intra.copy()
+    df_average['MJD'].values = np.around(df_average['MJD'].values.astype(np.float), decimals=0)

```

Comment:
I debugged this line directly. The input alert data is heterogeneous, hence it can cause trouble. The safest way is to use:
```python
df_average['MJD'] = df_average['MJD'].apply(lambda x: np.around(x, decimals=0))
```